apps/frontend/src/App.tsx

```
@@ -0,0 +1,121 @@
+      <div className="ticks"></div>
+
+      <section id="next-steps">
+        <div id="docs">
+          <svg className="icon" role="presentation" aria-hidden="true">
+            <use href="/icons.svg#documentation-icon"></use>
+          </svg>
+          <h2>Documentation</h2>
+          <p>Your questions, answered</p>
+          <ul>
+            <li>
+              <a href="https://vite.dev/" target="_blank">
```

Biome (2.4.6)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

(lint/security/noBlankTarget)